README.md

```diff
@@ -4,12 +4,17 @@ Welcome to General Services Administration (GSA) Information Security Engineerin
 
 ### What are these benchmarks?
 
-The ISE publishes security guides for various systems at the GSA. For more information, please refer to the published guide on insite.gsa.gov.  
+The ISE publishes security guides for various systems at the GSA. For more information, please refer to the published guide on insite.gsa.gov.
 
-### Available Tools
+### Reference
 
+* [DevSecOps example](https://github.com/GSA/devsecops-example)
+    * This is an example implementation of the DevSecOps pipeline, showing best practices for incorporating the other resources linked below
```

Comment:
I welcome feedback on this wording.

Reply:
Ok. I'm going to update the Readme as a whole with all the links from walkers meeting and clarifying a few items. I'll incorporate this in it.

Reply:
Responding with my GSA account....Ok. I'm going to update the Readme as a whole with all the links from walkers meeting and clarifying a few items. I'll incorporate this in it.

Reply:
I'm good with it...